### ISSUE TITLE
Google Analytics isn't working for homepage/root

### ISSUE THREAD

**Issue by taylordowns2000** [OpenFn/OpenFn-Site#165]
openfn.org isn't sending any data to google analytics. all the other pages in the site are working just fine. We're using angulartics (http://luisfarzati.github.io/angulartics/) which claims that it does all route tracking by default. It's a "one line setup" which involves injecting it into our app dependencies and then just removing a single line of google analytics code.

A while ago I fixed this by having a redirect to openfn.org/welcome for the root directory, but that's considered bad practice. It's as if Angulartics (which tracks all routes) doesn't view the ROOT as a ROUTE. Is that possible?

**Comment by taylordowns2000** [comment]
```
Remote Address:173.194.45.73:443
Request URL:https://www.google-analytics.com/collect?v=1&_v=j33&a=978269572&t=pageview&_s=1&dl=https%3A%2F%2Fstaging-openfn.herokuapp.com%2Fmappings&ul=en-us&de=UTF-8&dt=OpenFn&sd=24-bit&sr=1920x1080&vp=1871x473&je=1&fl=17.0%20r0&_u=AACAAEQBI~&jid=&cid=1833658453.1423040061&tid=UA-57118569-2&z=791487038
Request Method:GET
Status Code:200 OK
```

**Comment by stuartc** [comment]
@taylor, I think I've solved the issue. There was a smell around the split between the Rails and Angular routes.

The change is small but introduces a clearer separation, the way it used to work felt weird to me; wish I had thought of this before.

Anyway, I'd suggest checking this out locally and enable GA (I'll ping you in chat about how to do that).
I've tested quite a bit, and I can't find anything obviously broken. But what we need to watch out for is transitions between pages, "I was on x, I clicked y and I got the homepage instead".
